crates/sui-types/src/error.rs

```diff
@@ -657,6 +657,7 @@ impl SuiError {
                 (false, true)
             }
             SuiError::ObjectLockConflict { .. } => (false, true),
+            SuiError::CertificateRequiresQuorum => (true, true),
```

Comment:
if we get to this error, it means that something is broken in `stake_aggregator`, no?